README.md

```diff
@@ -78,16 +78,18 @@ az account show
 
 ### 2. Use inside a Development Container
 
+Using a development container will mean that all of the required software is installed for you. This will utilise WSL.
+
 #### Install the Pre-Requisite Software
 
-Install the following software on the machine you will perform the deployment from:
+Install the following software on the host machine you will perform the deployment from:
 
 >1. For Windows - [Windows Store Ubuntu 18.04 LTS](https://www.microsoft.com/store/productId/9N9TNGVNDL3Q)
 >2. [Docker Desktop](https://www.docker.com/products/docker-desktop)
 >3. [Visual Studio Code](https://visualstudio.microsoft.com/downloads/)
 >4. [Remote-Containers VS Code Extension](vscode:extension/ms-vscode-remote.remote-containers)
 
-#### For Windows - Configure WSL2 Backend for Docker Containers
+#### If you are using Docker Desktop 
```

Comment:
The text below talks about Ubuntu. It should really refer to WSL.
TBH, I wonder if *any* of this should be here. Would it be better to strip this out and refer the reader to a much better "how to use Dev Containers and WSL" article somewhere (assuming one exists).

Or, move all of this out of the README into a single "So...you're on Windows, WSL, and using a Dev Container" page where all of the "if you are..." stuff can be removed.